src/core/server/app/middleware/passport/strategies/verifiers/sso.ts

```diff
@@ -84,11 +84,6 @@ export async function findOrCreateSSOUser(
   token: SSOToken,
   now = new Date()
 ) {
-  if (!token.user) {
-    // TODO: (wyattjoh) replace with better error.
-    throw new Error("token is malformed, missing user claim");
```

Comment:
Why was this removed?

Reply:
The `token.user` is actually always available in a `SSOToken`, and is checked in the `supports` method elsewhere in the class.